Lib/test/test_minidom.py

```diff
@@ -1152,6 +1152,22 @@ def testEncodings(self):
 
         doc.unlink()
 
+    def testStandalone(self):
+        doc = parseString('<foo>&#x20ac;</foo>')
+        self.assertEqual(doc.toxml(),
+                         '<?xml version="1.0" ?><foo>\u20ac</foo>')
+        self.assertEqual(doc.toxml(standalone=None),
+                         '<?xml version="1.0" ?><foo>\u20ac</foo>')
+        self.assertEqual(doc.toxml(standalone=True),
+            '<?xml version="1.0"  standalone="yes"?><foo>\u20ac</foo>')
```

Comment:
There shouldn't be a double space before `standalone`. It's unfortunate that there is a space after the version even without further parameters, but then, maybe there should just be a space after each of the parameters?
In any case: we should keep the same serialisation as before for all cases without `standalone`, and make the ones with the new parameter consistent with what was there before.

Reply:
@scoder the double space there looks very much distracting, one the other hand adding a space after each of the parameters (i.e. `encoding`) causes failures in some of the past tests.

How about doing something like this:
```python
if encoding and standalone:
    standalone = ' ' + standalone
```